docs/otlp/getting-started-console/README.md

```diff
@@ -0,0 +1,126 @@
+# Getting Started with OpenTelemetry Protocol (OTLP) in 5 Minutes - Console Application
+
+If you haven't already, download and install the [.NET
+SDK](https://dotnet.microsoft.com/download) and
+[Docker](https://www.docker.com/) on your computer.
+
+Install and run the [Standalone .NET Aspire
+dashboard](https://learn.microsoft.com/dotnet/aspire/fundamentals/dashboard/standalone)
+using Docker:
+
+```sh
+docker run --rm -it -p 18888:18888 -p 4317:18889 -p 4318:18890 -d --name aspire-dashboard mcr.microsoft.com/dotnet/aspire-dashboard:latest
```

Comment:
When I follow these instructions and then browse to the .NET Aspire
dashboard (eg `http://localhost:18888/`) to view my telemetry, I get:

![image](https://github.com/user-attachments/assets/3f810235-2d5a-44d4-916c-73d33fa23df1)

I know how to get around this, but I'm not sure someone new would be able to figure it out. I think we should either:

1. Set the switch to disable the log in.
-OR-
2. Describe what someone needs to do to log in to the dashboard

Reply:
I updated the guides to disable auth. Also added a "caution" note about it. LMK